meta/main.yml

```diff
@@ -3,13 +3,37 @@ galaxy_info:
   author: Shane Frasier
   description: Ansible role for installing cisagov/code-gov-update
   company: CISA Cyber Assessments
+  galaxy_tags: []
   license: CC0
-  min_ansible_version: 2.0
+  # With the release of version 2.10, Ansible finally correctly
+  # identifies Kali Linux as being the Kali distribution of the Debian
+  # OS family.  This simplifies a lot of things for roles that support
+  # Kali Linux, so it makes sense to force the installation of Ansible
+  # 2.10 or newer.
+  min_ansible_version: 2.10
   platforms:
+    # Amazon Linux 2 doesn't appear to provide docker
+    # - name: Amazon
+    #   versions:
+    #     - 2
     - name: Debian
       versions:
         - stretch
-  galaxy_tags: []
+        - buster
+        # Kali linux isn't an option here, but it is based on
+        # Debian Testing:
+        # https://www.kali.org/docs/policy/kali-linux-relationship-with-debian
+        - bullseye
+    - name: Fedora
+      versions:
+        - 32
+        - 33
+    - name: Ubuntu
+      versions:
+        - xenial
+        - bionic
+        - focal
+  role_name: docker
```

Comment:
`role_name` should be `code_gov_update`.

Reply:
See commit 804f7c8.